src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Area;
+import core.basesyntax.Draw;
+import core.basesyntax.Figure;
+
+public class Circle extends Figure implements Draw, Area {
```

Comment:
Тобі не потрібно імплементувати інтерфейси які ти вже імплементував в батьківському класі
```suggestion
public class Circle extends Figure {
```